pkg/go/transformer/mod-to-json.go

```diff
@@ -8,10 +8,22 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+type ModFileStringProperty struct {
+	Value  string `json:"value"`
+	Line   int    `json:"line"`
+	Column int    `json:"column"`
```

Comment:
Should we line and column be:

```
Line: {
  Start: ...,
  End: ...
},
Column: {
  Start: ...,
  End: ...
},
```

Reply:
It should, but we only have starting information using the current yaml package and no real way to try and calculate it :/

It might be worthwhile evaluating other yaml packages in the go ecosystem to improve our reporting here

Reply:
Oh that's a bummer.

It's OK we can deal with it later.

I was thinking of the case where you have a comment on the line and don't want it to show as a squiggly line, but that's very far down the prio list. Let's keep as is for now until we find something better